### PR TITLE
chore: changelog v49.10.0

### PR DESCRIPTION
**Pull request by jag-sfdc**
### What does this PR do?
chore: changelog v49.10.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
+# 49.10.0 - September 16, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-soql
+
+- Fix a compile error by adding type for mkdirp ([PR #2524](https://github.com/forcedotcom/salesforcedx-vscode/pull/2524))
```
Comment (xyc) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]: This might not be relevant for users

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+
+## Added
+
+#### salesforcedx-vscode-soql
+
+- Soql Builder - Transform soql to ui and back to soql ([PR #2501](https://github.com/forcedotcom/salesforcedx-vscode/pull/2501))
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md:13]: Do we know if the SOQL Builder is ready for users yet? My guess is no and that we might not want to include these PRs in the change log yet.
@lcampos

Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md:7]: That is correct, we should not add anything related to soql until end of October

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
+# 49.10.0 - September 16, 2020
+
+## Added
+
+#### salesforcedx-vscode-soql
+
+- SOQL SObject Integration ([PR #2488](https://github.com/forcedotcom/salesforcedx-vscode/pull/2488))
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]: This piece and the header above it still needs to be removed. The rest looks good to me.

**Review by lcampos — APPROVED** [review]

**Review by sfsholden — APPROVED** [review]

**Pull request merged**